package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "typedoc-default-themes",
   "description": "Default themes for TypeDoc.",
-  "version": "0.4.1",
+  "version": "0.4.2",
```

Comment:
Sorry, you are right--since this only matters if you reference this repo as a submodule/directly, it doesn't need a version change--I'll revert it.